content/applications/finance/accounting/taxation/taxes/cash_basis_taxes.rst

```diff
@@ -1,95 +1,117 @@
-=======================
-Manage cash basis taxes
-=======================
-
-The cash basis taxes are due when the payment has been done and not at
-the validation of the invoice (as it is the case with standard taxes).
-Reporting your income and expenses to the administration based on the
-cash basis method is legal in some countries and under some conditions.
-
-Example : You sell a product in the 1st quarter of your fiscal year and
-receive the payment the 2nd quarter of your fiscal year. Based on the
-cash basis method, the tax you have to pay to the administration is due
-for the 2nd quarter.
-
-How to configure cash basis taxes ? 
-------------------------------------
-
-You first have to activate the setting in
-:menuselection:`Accounting --> Configuration --> Settings --> Allow Tax Cash Basis`.
-You will be asked to define the Tax Cash Basis Journal.
-
-.. image:: cash_basis_taxes/cash_basis_taxes01.png
-    :width: 5.04688in
-    :height: 0.79688in
+================
+Cash basis taxes
+================
+
+Cash basis taxes are due when the payment is made, as opposed to standard taxes that are due when
+the invoice is confirmed. Reporting your income and expenses to the administration based on the cash
+basis method is mandatory in some countries and under some conditions.
```

Comment:
Cash basis taxes are due when the payment is made, as opposed to standard taxes that are due when
the invoice is confirmed. Reporting your income and expenses to the government based on the cash
basis method is mandatory in some countries and under some conditions.